services/viva-ms/lambdas/createVivaCase.js

```diff
@@ -325,3 +332,12 @@ async function getLastUpdatedCase(PK, provider) {
 
   return sortedCases?.[0] || {};
 }
+
+function appendEncryptionAttributes(formAttributes, mainApplicantPno, coApplicantPno) {
```

Comment:
argument formAttributes could be any object, so i would rather se this renamed to `object` or `collection` since we do not know if the passed argument is formAttributes or not. We could check this but do not feel that there is any need for this :)